lib/helpers/multimedia/sort-images.js

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I've created this new folder to start to extract the function which are inside lib/jsonapi-response.js. I think it's a bit more readable and we can also unit test this functions. Happy to put it back to jsonapi-response file if we consider that the function only be used there, any recommendation @Danwhy?